components/pedal-card-preview/index.js

```diff
@@ -14,25 +16,36 @@ import {
 } from "./pedal-card-preview.styled";
 
 const PedalCardPreview = ({ pedalData }) => {
+  const router = useRouter();
+
+  const handleImageClick = () => {
+    router.push(`/detail/${pedalData.id}`);
+  };
+
   return (
     <PedalPreview>
       <PedalTitle>
-        <PedalName>{pedalData.NAME}</PedalName>
+        <PedalName>{pedalData.name}</PedalName>
         <PedalManufacturer>{pedalData.MANUFACTURER}</PedalManufacturer>
       </PedalTitle>
 
       <CategoryWrapper>
         {pedalData.CATEGORY.map((category, index) => (
-          <div key={index}>{category}</div>
+          <p key={index}>{category}</p>
```

Comment:
It is not a good practice to use the `index` as the `key` prop. Maybe the `category` itself would be unique enough? Are there more than the same category in the array?

Reply:
yes. But as this is an array in an object. Because sometimes a pedal is assigned one category and sometimes multiple, i have to view these in this way. i tried different ways without index and failed, so i kept it in for now.